addons/mrp/models/stock_orderpoint.py

```diff
@@ -104,7 +104,12 @@ def _quantity_in_progress(self):
         bom_manufacture = self.env['mrp.bom']._bom_find(orderpoints_without_kit.product_id, bom_type='normal')
         bom_manufacture = self.env['mrp.bom'].concat(*bom_manufacture.values())
         productions_group = self.env['mrp.production']._read_group(
-            [('bom_id', 'in', bom_manufacture.ids), ('state', '=', 'draft'), ('orderpoint_id', 'in', orderpoints_without_kit.ids)],
+            [
+                ('bom_id', 'in', bom_manufacture.ids),
+                ('state', '=', 'draft'),
+                ('orderpoint_id', 'in', orderpoints_without_kit.ids),
+                ('id', 'not in', self.env.context.get('ignore_mo_ids', [])),
+            ],
```

Comment:
To me we could add into `_quantity_in_progress` a search on the raw material stock move that will be confirmed in `post_process_scheduler` as well to update the quantity to order. 
For now there is only a search to update finished products quantities